config/environments/test.rb

```diff
@@ -34,7 +34,13 @@
   config.action_mailer.delivery_method = :test
 
   # Print deprecation notices to the stderr.
-  config.active_support.deprecation = :stderr
+  config.active_support.deprecation = lambda do |message, _backtrace|
+    if message =~ /DEPRECATION WARNING: .* HTTP request methods will accept only/
+      # ignore ... we'll have to deal with that later ...
```

Comment:
I'll fix this up after the merge, but would create too much diff atm ... and I want a clean test run :)